### DISCUSSION TITLE
Modern alternatives

### DISCUSSION THREAD

**Discussion by PedroHLC** [osdlyrics/osdlyrics#130] [Show and tell]
Are you guys aware that there is a new Linux-based, GTK4/Wayland-ready, Rust-written alternative to OSDLyrics?: @poly000's https://github.com/poly000/waylyrics

Maybe we can peacefully retire OSDLyrics. Cc @kbsky

**Comment by kbsky** [comment]
Wow that is really cool! Seems to be extremely young though so it's not time to retire OSDLyrics just yet :) I also seem to understand it's Wayland-only, unfortunately I'm using Cinnamon so no Wayland for me yet. Still it could be a very good base to expand on and bring to parity with OSD Lyrics feature-wise.

**Reply by mokurin000** [reply]
oh.. it's true that waylyrics is in very early stage, but it's name may be confusing.
waylyrics uses gtk's GTKSurface to allow click-through, thus it's not limited to wayland
waylyrics do not automatically set itself as top-level window, though some tricks related for X11 is already avaliable

I am neither familiar with GTK nor CSS, so a play animation may stay in the TODO list for few time
In 0.3.13, we have support for LRCLib
This might be better than Netease/QQMusic, for non-chinese users

**Reply by kbsky** [reply]
Good to hear it's not limited to Wayland! I should give it a try then.

**Comment by VlKlNGS** [comment]
Huh, first thing I see when I check the link is chinese writing, no way am I touching that!! You do you...

**Reply by mokurin000** [reply]
https://github.com/waylyrics/waylyrics/blob/master/README.en.md
README in English is available though,
waylyrics is mainly integrated with online services from China. Support for musixmatch and spotify are not yet implemented. You may try Sunamu

**Reply by VlKlNGS** [reply]
> waylyrics is mainly integrated with online services from China.

That is exactly my point, I don't care that they have an english translation, I am never gonna install(not knowingly anyway) a chinese piece of software on my pc! 
As it is it makes me sick to my stomach that a lot of the stuff I buy I can't find alternatives and am forced to buy made in china things. But I am doing my best to distance myself...
